e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-        if maxval < 10.0:
-            fmt = "%5.2f"
-            pad = 25
-        elif maxval < 100.0:
-            fmt = "%5.1f"
-            pad = 25
-        else:
-            fmt = "%6.1f"
-            pad = 30
-        cbar.set_ticks(levels[1:-1])
-        labels = [fmt % level for level in levels[1:-1]]
-        cbar.ax.set_yticklabels(labels, ha="right")
-        cbar.ax.tick_params(labelsize=9.0, pad=pad, length=0)
-
-    # Min, Mean, Max
-    fig.text(
-        panel[n][0] + 0.35,
-        panel[n][1] + 0.225,
-        "Max\nMean\nMin",
-        ha="left",
-        fontdict=plotSideTitle,
-    )
-    fig.text(
-        panel[n][0] + 0.45,
-        panel[n][1] + 0.225,
-        "%.2f\n%.2f\n%.2f" % stats[0:3],
-        ha="right",
-        fontdict=plotSideTitle,
-    )
-
-    # RMSE, CORR
-    if len(stats) == 5:
-        fig.text(
-            panel[n][0] + 0.35,
-            panel[n][1] + 0.0,
-            "RMSE\nCORR",
-            ha="left",
-            fontdict=plotSideTitle,
-        )
-        fig.text(
-            panel[n][0] + 0.45,
-            panel[n][1] + 0.0,
-            "%.2f\n%.2f" % stats[3:5],
-            ha="right",
-            fontdict=plotSideTitle,
-        )
-
-
-def plot(reference, test, diff, metrics_dict, parameter):
-    # Create figure, projection
-    fig = plt.figure(figsize=parameter.figsize, dpi=parameter.dpi)
-
-    # Create projection
-    if parameter.var_region.find("N") != -1:
-        pole = "N"
-        proj = ccrs.NorthPolarStereo(central_longitude=0)
-    elif parameter.var_region.find("S") != -1:
-        pole = "S"
-        proj = ccrs.SouthPolarStereo(central_longitude=0)
-
-    # First two panels
-    min1 = metrics_dict["test"]["min"]
-    mean1 = metrics_dict["test"]["mean"]
-    max1 = metrics_dict["test"]["max"]
-    if test.count() > 1:
-        plot_panel(
-            0,
-            fig,
-            proj,
-            pole,
-            test,
-            parameter.contour_levels,
-            parameter.test_colormap,
-            [parameter.test_name_yrs, parameter.test_title, test.units],
-            parameter,
-            stats=(max1, mean1, min1),
-        )
-
-    min2 = metrics_dict["ref"]["min"]
-    mean2 = metrics_dict["ref"]["mean"]
-    max2 = metrics_dict["ref"]["max"]
-
-    if reference.count() > 1:
-        plot_panel(
-            1,
-            fig,
-            proj,
-            pole,
-            reference,
-            parameter.contour_levels,
-            parameter.reference_colormap,
-            [
-                parameter.ref_name_yrs,
-                parameter.reference_title,
-                reference.units,
-            ],
-            parameter,
-            stats=(max2, mean2, min2),
-        )
-
-    # Third panel
-    min3 = metrics_dict["diff"]["min"]
-    mean3 = metrics_dict["diff"]["mean"]
-    max3 = metrics_dict["diff"]["max"]
-    r = metrics_dict["misc"]["rmse"]
-    c = metrics_dict["misc"]["corr"]
-
-    if diff.count() > 1:
```

Comment:
In the refactor, this check is combined with the ref check:
`e3sm_diags/plot/polar_plot.py`
```
if da_ref is not None and da_diff is not None:
```